.gitignore

```diff
@@ -0,0 +1,35 @@
+# Created by .ignore support plugin (hsz.mobi)
```

Comment:
This file needs to go completely.  I don't want a .gitignore until the build system is _really_ solidified.